packages/office-ui-fabric-react/src/components/Icon/Icon.types.ts

```diff
@@ -65,4 +67,9 @@ export interface IIconProps extends React.HTMLAttributes<HTMLElement> {
    * @memberOf IIconProps
    */
   imageProps?: IImageProps;
+
+  /**
+   * If rendering an image icon, this function callback will be invoked in the event loading the image errors.
+   */
+  imageErrorAs?: (image: IImageProps) => any;
```

Comment:
instead of `any`, can you make this type safe? Let me see;

```tsx
 React.StatelessComponent<IImageProps> | React.ComponentClass<IImageProps>
```
I *think* that's it?